src/main/java/edu/neu/khoury/cs5500/dijkstraspizza/model/Pizza.java

```diff
@@ -1,65 +1,64 @@
 package edu.neu.khoury.cs5500.dijkstraspizza.model;
```

Comment:
Note that we will probably need a unit test for this class now for code coverage. Alternatively, you could use lombok for everything except the constructors.